<?xml version="1.0"?>

<launch>
    
    <node pkg="multi_pc2_comb" type="multi_pc2_comb_node" name="multi_pc2_comb_node" output="screen">
        
        <param name="sub_topic_name_1" value="/ls_left/lslidar_point_cloud"/>
        <param name="sub_topic_name_2" value="/ls_right/lslidar_point_cloud"/>
        <param name="pub_topic_name" value="/lslidar_point_cloud_combined"/>
        
        <param name="rotation_x_1" value="0"/>
        <param name="rotation_y_1" value="0"/>
        <param name="rotation_z_1" value="0"/>
        <param name="offset_x_1" value="0"/>
        <param name="offset_y_1" value="0"/>
        <param name="offset_z_1" value="0"/>
        
        <param name="rotation_x_2" value="0"/>
        <param name="rotation_y_2" value="0"/>
        <param name="rotation_z_2" value="0"/>
        <param name="offset_x_2" value="0"/>
        <param name="offset_y_2" value="0"/>
        <param name="offset_z_2" value="0"/>
        
        <param name="show_points_size" value="true"/>
        
    </node>

</launch>
